packages/openwrt/etc/init.d/edge

```diff
@@ -14,9 +14,10 @@ start_service() {
 	procd_close_instance
 }
 
-stop_service()
+reload_service()
 {
-	service_stop $PROG
+	stop
+	start
```

Comment:
You appear to have removed the stop_service() completely, is that expected?

Reply:
Yes, procd will automatically stop the program by sending singal `SIGTERM`.
See https://forum.openwrt.org/t/procd-shutdown-issues-questions/33759/2